daml-lf/encoder/src/main/scala/com/digitalasset/daml/lf/archive/testing/EncodeV1.scala

```diff
@@ -800,7 +803,10 @@ private[daml] class EncodeV1(minor: LV.Minor) {
       builder.setParamInternedStr(stringsTable.insert(interface.param))
       builder.accumulateLeft(interface.choices.sortByKey)(_ addChoices _)
       builder.accumulateLeft(interface.methods.sortByKey)(_ addMethods _)
-      builder.accumulateLeft(interface.requires)(_ addRequires _)
+      if (interface.requires.nonEmpty) {
+        assertSince(LV.Features.choiceObservers, "DefInterface.requires")
```

Comment:
👁️📎👁️: I think you meant
```suggestion
        assertSince(LV.Features.extendedInterfaces, "DefInterface.requires")
```

Reply:
Good catch ! 

My tests should have catch that.